applications/nrf5340_audio/src/modules/display/display.c

```diff
@@ -0,0 +1,86 @@
+/*
+ * Copyright (c) 2023 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+ */
+
+#include <display/display.h>
+#include <zephyr/device.h>
+#include <zephyr/devicetree.h>
+#include <zephyr/drivers/display.h>
+#include <zephyr/drivers/gpio.h>
+#include <lvgl.h>
+#include <stdio.h>
+#include <string.h>
+#include <zephyr/kernel.h>
+#include <zephyr/zbus/zbus.h>
+#define LOG_LEVEL CONFIG_LOG_DEFAULT_LEVEL
+#include <zephyr/logging/log.h>
+#include <stdint.h>
+#include "button_tab.h"
+
+#define MY_PRIORITY 5
+LOG_MODULE_REGISTER(app);
+
+int test;
+enum button_action {
+	BUTTON_PRESS,
+	BUTTON_ACTION_NUM,
+};
+struct button_msg {
+	uint32_t button_pin;
+	enum button_action button_action;
+};
+
+ZBUS_CHAN_DECLARE(button_chan);
+
+void create_timestamp_label(lv_obj_t *current_screen)
+{
+	static uint32_t count;
+	char count_str[11] = {0};
+	lv_obj_t *count_label;
+	count_label = lv_label_create(current_screen);
+	lv_obj_align(count_label, LV_ALIGN_BOTTOM_MID, 0, 0);
+	while (1) {
+		if ((count % 100) == 0U) {
+			sprintf(count_str, "%d", count / 100U);
+			lv_label_set_text(count_label, count_str);
+		}
+		lv_task_handler();
+		++count;
+		k_sleep(K_MSEC(10));
+	}
+}
+K_THREAD_STACK_DEFINE(update_volume_level_thread_STACK, CONFIG_MAIN_STACK_SIZE);
+
+void display_init()
```

Comment:
As this function can actually fail at runtime (`device_is_ready()` may fail) it should return an error value so the application knows when it's not succesful